pkg/codegen/testing/test/program_driver.go

```diff
@@ -25,6 +25,10 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
 )
 
+const (
+	TRANSPILED_EXAMPLES_DIR = "transpiled_examples/"
+)
+
```

Comment:
We shouldn't introduce the constant, and then hardcode it everywhere. I would add a helper function like this:
```suggestion

func transpiled(dir string) string {
    return filepath.join(TRANSPILED_EXAMPLES_DIR, dir)
}
```

This way we can freely change the value of `TRANSPILED_EXAMPLES_DIR` and we only need to change it in one place.